src/nmodl.rs

```diff
@@ -732,10 +732,21 @@ fn nmodl_neuron_block(n: &Nmodl) -> Result<String> {
     }
 
     if n.kind == Kind::Density {
+        let mut ns = Vec::new();
         for ion in &ions {
             let ix = format!("i{ion}");
             if write.contains(&ix) && !n.known_ions.contains(ion) {
-                result.push(format!("  NONSPECIFIC_CURRENT {ix}\n"));
+                ns.push(ix);
+            }
+        }
+        match ns.as_slice() {
+            [] => {}
+            [name] => {
```

Comment:
That's a nice way to handle it, but maybe you use the simpler approach
of always collapsing and accept one additional `LOCAL` (which will never impact binary output, since C++ will splat this into registers anyhow)

Reply:
Like making the `ihcn` etc.. write into  a `LOCAL ihcn` and and then summing up `i = g*(V-E) + ihcn + ik + ...` ?

Reply:
Or just always using `i` as NONSPECIFIC current name

Reply:
I was thinking about the latter, but your idea is also good!